src/components/BarBalance/styles.styl

```diff
@@ -1,3 +1,4 @@
 .BarBalance
     font-size 1.125rem
     color var(--primaryTextColor)
+    margin-right 0.5rem
```

Comment:
The idea was not to use `spacing_values` from UI to deal with the gutters ?

Reply:
No, I had simply not thought of using spacing_values :) Fixed, thanks !